server/server.js

```diff
@@ -116,7 +116,7 @@ var Server = IgeClass.extend({
 		self.serverStartTime = new Date();// record start time
 		global.isDev = ige.env == 'dev' || ige.env == 'local' || ige.env === 'standalone' || ige.env === 'standalone-remote';
 		global.myIp = process.env.IP;
-		global.beUrl = self.config.BE_URL;
+		global.beUrl = self.config.BE_URL || 'http://gs_manager.modd.io';
```

Comment:
This is redundant, no?

Reply:
In what scenario is there not a value for `self.config.BE_URL` is the question I guess. If there isn't a value, I think the problem is elsewhere.